docs/hardware/devices/heltec/peripherals.mdx

```diff
@@ -0,0 +1,90 @@
+However, it's important to note that the GPS module increases the power demand of your node. We'll address this by detailing how to incorporate a switch or an NPN 2N2222 transistor into your setup. This enables firmware-controlled power management, conserving battery life without sacrificing functionality.
+
+It is important to note that the GPS module increases the power demand of the node. This guide details the incorporation of a switch or an NPN 2N2222 transistor to enable firmware-controlled power management, conserving battery life without compromising functionality.
+
+### Benefits
+- **GPS Capabilities**: Provides the node with the ability to determine its location with high precision, which is invaluable for tracking, mapping, and various other applications requiring location data.
+- **Real-Time Clock (RTC)**: Ensures accurate timekeeping on the mesh network without relying on external time sources such as the internet or a connected smartphone.
+
+### Power Consumption Considerations
+The GT-U7 module is known for its high power consumption, which can potentially shorten the battery lifespan of the node. To mitigate this, two approaches are recommended:
+ - **Manual Switch**: A simple on/off switch for the GPS module, allowing for manual power management.
+ - **NPN 2N2222 Transistor**: Facilitates automatic power control through the firmware, enabling the device to turn off the GPS module based on specific conditions or after a set period.
+
+### Materials Needed
+ - Heltec ESP32 V3 board
+ - GT-U7 GPS Module
+ - NPN 2N2222 Transistor
+ - Wires and soldering equipment
+ - (Optional) Switch for manual power control
+
+### Instructions
+
+ 1. Solder a cable from the TXD slot on the GPS module to GPIO 48 on Heltec board. (You may choose your own GPIO pin)
+ 2. Solder a cable from the RXD slot on the GPS module to GPIO 47 on Heltec board. (You may choose your own GPIO pin)
+ 3. Solder a cable from the GND slot on the GPS module to GND pin on Heltec board.
+ 4. Solder a cable from left leg of NPN 2N2222 Transistor to VCC skit on GPS module.
+ 5. Solder a cable from Right leg of NPN 2N2222 Transistor to 3V/5V pin on Heltec board.
+ 6. Solder a cable from Middle leg of NPN 2N2222 Transistor to GPIO 48 of Heltec board. (You may choose your own GPIO pin)
+ 7. Go to Meshtastic app > Radio Configurations > Position
+ 8. Set GPS_RX_PIN to 48 (This will communicate to the TXD slot on the GPS)
+ 9. Set GPS_TX_PIN to 47 (This will communicate to the RXD slot on the GPS)
+ 10. Set PIN_GPS_EN to 46 (This will allow the meshtastic firmware to turn off the power on the GPS board with the user button of the Heltec Board by pressing it 3 times)
+
+## Wiring Diagram
```

Comment:
Sorry, just noticed one last thing, this, and the headings below it, should all be >3 as they are all children of GPS Module which is a 2